apps/examples/hello/hello_main.c

```diff
@@ -68,5 +69,11 @@ int hello_main(int argc, char *argv[])
 #endif
 {
 	printf("Hello, World!!\n");
+#ifdef CONFIG_PM
```

Comment:
Let's make new example for pm because this is the hello world example.

Reply:
if CONFIG_PM is enabled, we want to block any PM transition till the time the board is completed booted and settled. I think this locking of PM should be applicable to all examples where CONFIG_PM can be enabled. Let me know your views.

Reply:
Hello world does not have any loop. It just prints hello and quit. I think that adding pm code in hello is redundant.

Reply:
Made a new example for power